components/_patterns/02-molecules/date-bars/date-bar.twig

```diff
@@ -0,0 +1,47 @@
+{#
+/**
+ * @file
+ * Event address bar for displaying date and time information for events.
+ * 
+ * Variables:
+ * # event_date_start — The start date/time.
+ * # event_date_end — The end date/time.
+ * # event_time_range — String with the time range.
+ */
+#}
+
+{% set date_bar_base_class = 'date-bar' %}
+
+{% if date(event_date_end) < date() %}
+  {% set date_bar_modifiers = ['ended'] %}
+{% endif %}
+
+<div {{ bem(date_bar_base_class , (date_bar_modifiers), date_bar_blockname) }}>
+  {% if date(event_date_end) < date() %}
+    The event has ended
+  {% elseif event_date_end|date('Y-m-d') > event_date_start|date('Y-m-d') %}
```

Comment:
I don't think this is working the way you expect. It is showing different variations in Pattern Lab, but the start date used in the "Range" variation is in the future... so "Open now" isn't true.

It seems like we have the following variations
- Ended (end date is in the past)
- Single day/time (range) (future start date)
- Spans multiple days (and should show both days/times?) (future start date)
- Spans multiple days ("Open now - " end date) (past start date, future end date)

... I think that's it?

Reply:
I think this means we need another variation. One to show the range with both dates in the future, and one to show the range of a currently open event.

Reply:
I think you're right @ModulesUnraveled. I'll follow up with the designers in Figma about a time range variation.

Reply:
I've updated the Jira ticket and left some questions for the designers in the comments in Figma here: https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Working-Component-Library?node-id=674%3A123

Reply:
Looks like, we should actually be using the `<time>` element for the best SEO https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
I implemented this based on Slack/Zoom conversations, so it's resolved.